easybuild/easyconfigs/o/OpenSlide/OpenSlide-3.4.1-foss-2019a-largefiles.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenSlide'
+version = '3.4.1'
+versionsuffix = '-largefiles'
+
+homepage = 'https://openslide.org/'
+description = """OpenSlide is a C library that provides a simple interface to
+read whole-slide images (also known as virtual slides)."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
I missed this until looking at vips, and noticing the chat that another person had put it at GCCcore level. Why was this moved into foss? GCC level at most i could maybe understand.